### PR TITLE
vizlab footer update

### PR DESCRIPTION
**Pull request by mwernimont**
New styles and removing some unneeded and confusing files, and a couple lines of code.

### CODE REVIEW

**Comment by mwernimont** [comment]
boom, +.08 baby

File: inst/templates/footer.mustache [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
-      <a href="https://usgs.gov" target="_blank">
-        {{{usgsLogo}}}
-      </a>
+<div class="footerContentAligner">
+
+    <div id="vizlabPromo">
+      <p>This interactive data visualization was created by 
+      <a href="https://owi.usgs.gov/vizlab/" target="_blank">
+        <strong>USGS VIZLAB</strong></p>
+      	<img src="images/atom.svg" alt="Vizlab Atom">
```
Comment (jiwalker-usgs) [inline inst/templates/footer.mustache]: This image tag should be closed, also I'm wondering if this image should be added via the normal publish.svg process.

File: inst/templates/footer.mustache [hunk 2]
```diff
@@ -1,63 +1,63 @@
...
-        <p>About Us</p>
-      </a>
-      <a href="{{github-url}}" target="_blank" onclick="ga('send', 'event', 'outbound', 'clicked on repo')">
-        <p>Get the Code</p>
-      </a>
-      <a href="https://owi.usgs.gov/" target="_blank">
-        <p>Office of Water Information</p>
+    <div id="github-contact">
+      <a class="hotfix" href="#" target="_blank" onclick="ga('send', 'event', 'outbound', 'clicked on repo')">
+        <img src="images/github.svg" alt="github icon">
```
Comment (jiwalker-usgs) [inline inst/templates/footer.mustache]: similar comment to above

File: inst/templates/footer.mustache [hunk 3]
```diff
@@ -1,63 +1,63 @@
...
-      <a href="https://owi.usgs.gov/vizlab/" target="_blank">
-        <p>About Us</p>
-      </a>
-      <a href="{{github-url}}" target="_blank" onclick="ga('send', 'event', 'outbound', 'clicked on repo')">
-        <p>Get the Code</p>
-      </a>
-      <a href="https://owi.usgs.gov/" target="_blank">
-        <p>Office of Water Information</p>
+    <div id="github-contact">
+      <a class="hotfix" href="#" target="_blank" onclick="ga('send', 'event', 'outbound', 'clicked on repo')">
```
Comment (jiwalker-usgs) [inline inst/templates/footer.mustache]: I think hotfix suggests that there should be a more permanent solution for this.

File: inst/templates/footer.mustache [hunk 4]
```diff
@@ -1,63 +1,63 @@
...
-  
-    <div class="usgsLogoFooterArea">
-      <a href="https://usgs.gov" target="_blank">
-        {{{usgsLogo}}}
-      </a>
+<div class="footerContentAligner">
+
+    <div id="vizlabPromo">
+      <p>This interactive data visualization was created by 
+      <a href="https://owi.usgs.gov/vizlab/" target="_blank">
```
Comment (jiwalker-usgs) [inline inst/templates/footer.mustache]: This should probably be templated in so it is easy to change if we move from owi.usgs.gov

**Comment by mwernimont** [comment]
Closed the img tags and renamed the hotfix class, its a pretty sound practice, can't remember why I called it that in the first place.

**Pull request merged**